go/pkg/entities/entities.go

```diff
@@ -0,0 +1,8 @@
+package entities
+
+import "berty.tech/yolo/v2/go/pkg/yolopb"
+
+type BuildListFilters struct {
```

Comment:
no, we should avoid adding more packages, especially for this

Reply:
@moul I moved the structure inside of the store pkg